### DISCUSSION TITLE
1.8.0.0

### DISCUSSION THREAD

**Discussion by georgefst** [haskell/haskell-language-server#3084] [General]
Evidently, the monthly release schedule has slipped. It's been more than three months!

I'm just curious - is there anything blocking a new release, or is it just a case of someone having the time to drive it (I'm not volunteering, unfortunately)?

**Comment by fendor** [comment]
In the monthly HLS meeting, it was discussed (maybe even decided?) that the monthly release is not viable at the moment. 
I think, we need to come up with a new release cycle, or with enough automation (which is my favourite) s.t. releases are not a lot of work.

**Comment by pepeiborra** [comment]
There are also plans to use the OpenCollective funds to pay for HLS releases, @michaelpj and the HF are driving it
